### PR TITLE
Please review changes

### PR DESCRIPTION
**Pull request by fsancheziohk**

### CODE REVIEW

**Comment by michaelpj** [comment]
Whoops, I just merged another PR that changes the Haskell references to use roles, so this has conflicts now. I recommend rebasing. While you're at it, can you squash your changes down into a single commit?

File: doc/tutorials/plutus-tx.rst [hunk 1]
```diff
@@ -6,53 +6,46 @@ What is Plutus Tx?
...
 The parts of the program that describe the on-chain code are still just
-Haskell, but they are compiled into :term:`Plutus Core` (instead of into the
-normal compilation target language) and we refer to them as :term:`Plutus Tx`
-("Tx" since they usually go into transactions).
+Haskell, but they are compiled into :term:`Plutus Core`, rather than into 
+the normal compilation target language. We refer to them as :term:`Plutus Tx`
+(where Tx indicates that these components usually go into transactions).
 
-.. warning::
+.. **warning::
```
Comment (michaelpj) [inline doc/tutorials/plutus-tx.rst:17]: I don't think this is right. This breaks the admonition?

File: doc/tutorials/plutus-tx.rst [hunk 2]
```diff
@@ -6,53 +6,46 @@ What is Plutus Tx?
...
 The key technique that we use to implement Plutus Tx is called *staged
-metaprogramming*. What that means is that the main Haskell program
-generates *another* program, in this case the Plutus Core program that
-will run on the blockchain. Plutus Tx is the mechanism that we use to
-write those programs. But because Plutus Tx is just part of the main
-Haskell program we can share types and definitions between the two.
+metaprogramming*, which means that the main Haskell program generates *another* program 
+(in this case, the Plutus Core program that will run on the blockchain). Plutus Tx is the 
+mechanism we use to write those programs, but since Plutus Tx is just part of the main
+Haskell program, we can share types and definitions between the two.
```
Comment (michaelpj) [inline doc/tutorials/plutus-tx.rst:27]: I'd suggest not reflowing the text if you can avoid it, it makes it harder to review changes...

(I've actually started writing with one sentence per line these days for that reason, but that's not to everyone's taste!)

File: doc/tutorials/plutus-tx.rst [hunk 3]
```diff
@@ -6,53 +6,46 @@ What is Plutus Tx?
...
 Template Haskell begins with *quotes*. A Template Haskell quote is a
 Haskell expression ``e`` inside special brackets ``[|| e ||]``. It has
 type ``Q (TExp a)`` where ``e`` has type ``a``. ``TExp a`` is a
 *representation* an expression of type ``a``, i.e. the syntax of the
-actual Haskell expression which was quoted. The quote lives in the type
+actual Haskell expression that was quoted. The quote lives in the type
 ``Q`` of quotes, which isn’t very interesting for us.
 
-.. note::
+.. **note::
```
Comment (michaelpj) [inline doc/tutorials/plutus-tx.rst:48]: Again, this breaks the admonition.

File: doc/tutorials/plutus-tx.rst [hunk 4]
```diff
@@ -102,33 +90,28 @@ Writing basic PlutusTx programs
    :start-after: BLOCK1
    :end-before: BLOCK2
 
-Here’s the most basic program we can write: one that just evaluates to
-the integer ``1``.
+This simple program just evaluates to the integer ``1``.
 
-.. note::
+.. **note::
```
Comment (michaelpj) [inline doc/tutorials/plutus-tx.rst:95]: Ditto

File: doc/tutorials/plutus-tx.rst [hunk 5]
```diff
@@ -102,33 +90,28 @@ Writing basic PlutusTx programs
...
 
-   We’ve included doctests to the examples that show the Plutus Core
-   that is generated from compilation. The syntax of Plutus Core will
-   look unfamiliar. This is fine, since it is the "assembly language"
-   and you won’t need to inspect the output of the compiler. However,
-   for our purposes it’s instructive to look at it to get a vague idea
-   of what’s going on.
+The examples that show the Plutus Core generated from compilation include doctests. The syntax of Plutus Core
```
Comment (michaelpj) [inline doc/tutorials/plutus-tx.rst:97]: This was indented so it goes in the admonition. Or did you mean to move this into the body text?

File: doc/tutorials/plutus-tx.rst [hunk 6]
```diff
@@ -228,31 +206,29 @@ Lifting values
...
-to do this called *lifting*. Lifting makes it easy to use the same types
-both inside your Plutus Tx program and in the external code that uses
-it.
+In our case, there is a slight complication: we want to make the argument
+and apply the function to it at *runtime*. Plutus Tx addresses this through *lifting*.
+Lifting enables the use of the same types, both inside your Plutus Tx program *and* in
+the external code that uses it.
 
-.. note::
+.. **note::
```
Comment (michaelpj) [inline doc/tutorials/plutus-tx.rst:224]: Ditto.

File: doc/tutorials/plutus-tx.rst [hunk 7]
```diff
@@ -267,20 +243,20 @@ it.
...
 
-We lifted the argument using the ``liftCode`` function. In order to use
-this, a type must have an instance of the ``Lift`` class. In practice,
+We lifted the argument using the ``liftCode`` function. To use this,
+a type must have an instance of the ``Lift`` class. In practice,
 you should generate these with the ``makeLift`` TH function from
 ``Language.PlutusTx.Lift``.
 
-.. note::
+.. **note::
```
Comment (michaelpj) [inline doc/tutorials/plutus-tx.rst:251]: Ditto.

**Comment by michaelpj** [comment]
Also, you can review what RTD generated from the link in the PR statuses - that's how I checked that the admonitions were indeed broken!

**Comment by fsancheziohk** [comment]
The tutorial was reviewed from a grammar & text flow perspective. Code lines remains intact.

**Comment by michaelpj** [comment]
Merged in https://github.com/input-output-hk/plutus/pull/2160

**Pull request closed without merging**